src/aiy/common.py

```diff
@@ -0,0 +1,228 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This library provides common drivers for the AIY projects.
+
+Drivers in this module requires GPIO.setmode(GPIO.BCM).
```

Comment:
If you call this in the code (eg after importing or in `__init__`) then it is enforced in code. `setmode` can safely be called multiple times with the same mode.